src/Sulu/Bundle/SecurityBundle/Entity/BaseUser.php

```diff
@@ -152,7 +152,7 @@ public function getUsername()
      */
     public function getFullName()
     {
-        return $this->getContact()->getFullName();
+        return $this->getUsername();
```

Comment:
Here I'm really not sure how to handle it as `getContact()` is only defined in `User` Entity where this function is also overwritten: https://github.com/sulu/sulu/blob/1.6.7/src/Sulu/Bundle/SecurityBundle/Entity/User.php#L242-L246